pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -279,11 +279,11 @@ stdenv.mkDerivation {
     else "";
 
   crossAttrs = {
-    shell = shell.crossDrv + shell.crossDrv.shellPath;
-    libc = stdenv.ccCross.libc;
-    coreutils = coreutils.crossDrv;
-    binutils = binutils.crossDrv;
-    cc = cc.crossDrv;
+    #shell = shell.crossDrv + shell.crossDrv.shellPath;
+    #libc = stdenv.ccCross.libc;
+    #coreutils = coreutils.crossDrv;
+    #binutils = binutils.crossDrv;
+    #cc = cc.crossDrv;
```

Comment:
Heh, this is me not finishing trying to figure whether they are needed. there's a lot of knots to untangle!